Libraries/react-native/react-native.js

```diff
@@ -121,6 +121,7 @@ const ReactNative = {
   get ColorPropType() { return require('ColorPropType'); },
   get EdgeInsetsPropType() { return require('EdgeInsetsPropType'); },
   get PointPropType() { return require('PointPropType'); },
+  get IndicatorSizePropType() { return require('IndicatorSizePropType'); },
```

Comment:
It can still be accessed by `ActivityIndicator.propTypes.size`, so yeah, unnecessary.